src/transaction/deploy_account.rs

```diff
@@ -194,7 +194,7 @@ impl DeployAccount {
             Rc<RefCell<ProgramCache<'_, ClassHash>>>,
         >,
     ) -> Result<TransactionExecutionInfo, TransactionError> {
-        if self.version != Felt252::ONE {
+        if !(self.version == Felt252::ONE || self.version == Felt252::THREE) {
             return Err(TransactionError::UnsupportedTxVersion(
                 "DeployAccount".to_string(),
                 self.version,
```

Comment:
The line below contains the valid transaction versions (which are used when displaying the unsupported version error), we should add the new version there too

Reply:
Good catch!